### PR TITLE
Add binding for ElementHandle.contentFrame

### LINKED ISSUE #23: Binding for ElementHandle.contentFrame()
This should be included in Puppeteer 1.2.0 which is expected to be released on March 15th.

[Docs](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#elementhandlecontentframe)

### PR DESCRIPTION
**Pull request by jihchi**
Fix #23 

it works with `puppeteer@next`.

### CODE REVIEW

File: src/ElementHandle.re [hunk 1]
```diff
@@ -49,3 +52,6 @@ external type_ :
 
 [@bs.send.pipe: t] [@bs.splice]
 external uploadFile : (~filePaths: array(string)) => Js.Promise.t(unit) = "";
+
+[@bs.send.pipe: t]
+external contentFrame : Js.Promise.t(Js.Nullable.t(Types.frameBase)) = "";
```
Comment (zploskey) [inline src/ElementHandle.re]: Should likely also be Js.null. We can use Js.nullToOption etc.

File: src/ElementHandle.re [hunk 2]
```diff
@@ -17,7 +20,7 @@ external selectAll : (~selector: string) => Js.Promise.t(array(t)) = "$$";
 external selectXPath : (~xpath: string) => Js.Promise.t(array(t)) = "$x";
 
 [@bs.send.pipe: t]
-external boundingBox : Js.Promise.t(Js.null(BoundingBox.t)) = "";
+external boundingBox : Js.Promise.t(Js.Nullable.t(BoundingBox.t)) = "";
```
[the remark this answers is not in the document]
Comment (jihchi) [reply src/ElementHandle.re]: Reverted in commit 51cb1c3

File: src/FrameBase.re [hunk 3]
```diff
@@ -14,7 +14,7 @@ external makeTagOptions :
 
 [@bs.send.pipe: t]
 external selectOne :
-  (~selector: string) => Js.Promise.t(Js.null(ElementHandle.t)) =
+  (~selector: string) => Js.Promise.t(Js.Nullable.t(ElementHandle.t)) =
```
[the remark this answers is not in the document]
Comment (jihchi) [reply src/FrameBase.re]: Reverted in commit 51cb1c3

File: src/ElementHandle.re (hunk 1 above)
Comment (jihchi) [reply src/ElementHandle.re]: Fixed in commit 51cb1c3

File: src/FrameBase.re (hunk 3 above)
[the remark this answers is not in the document]
Comment (jihchi) [reply src/FrameBase.re]: Reverted in commit 728c30d

**Pull request merged**